### PR TITLE
Scip 7

### PR DESCRIPTION
**Pull request by mueldgog**

### CODE REVIEW

**Comment by fserra** [comment]
windows fails... we should have test on windows before. Do you understand the error @mueldgog @mattmilten ?

**Comment by rschwarz** [comment]
While upgrading SCIP.jl (see https://github.com/SCIP-Interfaces/SCIP.jl/pull/149) I found that the old code that was looking for `scip.dll` on Windows now [fails to find it](https://ci.appveyor.com/project/rschwarz/scip-jl/builds/31872711#L94). Did the filename or path change for the Windows package?

**Comment by fserra** [comment]
interesting it seems the same problem occurs in pyscipopt

**Comment by fserra** [comment]
oh we rae in pyscipopt hahaa

**Comment by fserra** [comment]
ok, so if one install scip6 in windows its lib directory has the following files
```
scip.lib
soplex.lib
soplex-pic.lib
zimpl.lib
zimpl-pic.lib
```

on the other hand, scip7 has the following files
```
libscip.lib
soplex.lib
soplex-pic.lib
```

i don't know who tells the installer to look for scip.lib, but apparently now it should look for libsciip.lib (?)

**Comment by fserra** [comment]
Linux builds also fail, but I don't know why they are not shown here. We cannot extract the deb package because of dependencies

```
$ sudo apt-get update && sudo apt install -y ./SCIPOptSuite-$VERSION-Linux.deb
621Hit:1 http://apt.postgresql.org/pub/repos/apt xenial-pgdg InRelease
622Hit:2 http://security.ubuntu.com/ubuntu xenial-security InRelease
623Hit:3 http://archive.ubuntu.com/ubuntu xenial InRelease
624Hit:4 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
625Hit:5 http://archive.ubuntu.com/ubuntu xenial-backports InRelease
626Reading package lists...
627
628WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
629
630Reading package lists...
631Building dependency tree...
632Reading state information...
633Some packages could not be installed. This may mean that you have
634requested an impossible situation or if you are using the unstable
635distribution that some required packages have not yet been created
636or been moved out of Incoming.
637The following information may help to resolve the situation:
638
639The following packages have unmet dependencies:
640 scipoptsuite : Depends: libgfortran4 (>= 7) but it is not installable
641                Depends: libgsl23 but it is not installable
642                Depends: libgslcblas0 but it is not installable
643                Depends: liblapack3 but it is not going to be installed or
644                         liblapack.so.3
645E: Unable to correct problems, you have held broken packages.
646The command "sudo apt-get update && sudo apt install -y ./SCIPOptSuite-$VERSION-Linux.deb" failed and exited with 100 during .
647
648
```

what is the problem, are they too new?

**Comment by rschwarz** [comment]
> what is the problem, are they too new?

Yes, I think this is a problem of building SCIP on a newer version of Ubuntu.
This can be fixed by selecting `bionic` in [travis.yml](https://github.com/SCIP-Interfaces/SCIP.jl/pull/149/files#diff-354f30a63fb0907d4ad57269548329e3) .

**Comment by fserra** [comment]
ok thanks.

Do you know why the builds don't appear here? I only see the appveyor ones.

**Comment by fserra** [comment]
ok the travis test pass

File: appveyor.yml [hunk 1]
```diff
@@ -36,9 +36,9 @@ install:
   # - ps: $pass = $env:optipw | ConvertTo-SecureString -AsPlainText -Force
   # - ps: $cred = New-Object Management.Automation.PSCredential ($user, ($pass))
   # - ps: Invoke-WebRequest -Uri $uri -Credential $cred -OutFile 'scipopt-installer.exe'
-  - ps: wget http://scip.zib.de/download/release/SCIPOptSuite-6.0.2-win64-VS15.exe -outfile scipopt-installer.exe
+  - ps: wget http://scip.zib.de/download/release/SCIPOptSuite-7.0.0-win64-VS15.exe -outfile scipopt-installer.exe
   - scipopt-installer.exe /S /D=%SCIPOPTDIR%
-  - set PATH=%SCIPOPTDIR%\bin;%PYTHON%;%PATH%
+  - set PATH=%SCIPOPTDIR%\lib;%SCIPOPTDIR%\bin;%PYTHON%;%PATH%
```
Comment (mattmilten) [inline appveyor.yml:41]: Why would this be necessary?

Comment (fserra) [reply appveyor.yml:41]: is not i have to revert that

**Comment by mattmilten** [comment]
🍾 🎉 🎆 👍

**Comment by fserra** [comment]
we now see the travis builds too

**Comment by fserra** [comment]
@rschwarz the build on SCIP.jl should work with the newly uploaded scip windows executable. Can you give it a try plz?

**Pull request merged**